Pkgfile

```diff
@@ -11,9 +11,9 @@ vars:
   binutils_sha512: a37e042523bc46494d99d5637c3f3d8f9956d9477b748b3b1f6d7dfbb8d968ed52c932e88a4e946c6f77b8f48f1e1b360ca54c3d298f17193f3b4963472f6925
 
   # renovate: datasource=git-tags extractVersion=^releases/gcc-(?<version>.*)$ depName=git://gcc.gnu.org/git/gcc.git
-  gcc_version: 12.3.0
-  gcc_sha256: 949a5d4f99e786421a93b532b22ffab5578de7321369975b91aec97adfda8c3b
-  gcc_sha512: 8fb799dfa2e5de5284edf8f821e3d40c2781e4c570f5adfdb1ca0671fcae3fb7f794ea783e80f01ec7bfbf912ca508e478bd749b2755c2c14e4055648146c204
+  gcc_version: 13.1.0
```

Comment:
let's wait for more time, GCC 13 is too new, and lots of builds will break

Reply:
Done